examples/kcp/test/e2e/lowlevel_test.go

```diff
@@ -0,0 +1,69 @@
+package e2e
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	datav1alpha1 "github.com/kcp-dev/controller-runtime/examples/kcp/apis/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/errors"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+// TestWidgetController verifies that our ConfigMap behavior works.
+func TestWidgetController(t *testing.T) {
```

Comment:
This is duplicate with `controller_test.go` function too. I think this is wrong copy?